website_sale_product_sort/models/website.py

```diff
@@ -21,6 +21,6 @@ def _get_product_sort_criterias(self):
         selection="_get_product_sort_criterias",
         string="Sort products by",
         help="Default criteria for sorting products in the shop",
-        default="website_sequence desc",
+        default="website_sequence asc",
```

Comment:
This is important and I saw it isn't considered in OpenUpgrade for reversing the current ids :thinking:

Reply:
I think this change is related to this: https://github.com/odoo/odoo/blob/13.0/addons/website_sale/controllers/main.py#L154

Reply:
Yes, sure, this is the commit. The migration is right (I approved :slightly_smiling_face: ). I'm just commenting that a OpenUpgrade migration script would be needed additionaly to avoid issues in migrations.